test/manager_test.js

```diff
@@ -62,6 +62,16 @@ describe("Manager", function() {
         });
       });
 
+      it("compiles files that are imported when option is set", function(done) {
+        var spy = sinon.spy(manager.files['test/less/common.less'], 'compile');
+        manager.compileOptions.compileImports = true;
```

Comment:
Instead of accessing this directly (depending on implementation of `Manager`), let's just instantiate the manager again here.